virtcontainers/fc.go

```diff
@@ -579,6 +584,68 @@ func (fc *firecracker) fcSetVMBaseConfig(mem int64, vcpus int64, htEnabled bool)
 	return err
 }
 
+func (fc *firecracker) fcSetLogger() error {
```

Comment:
I don't like the `fc` prefix as it's redundant but I see there is already precedent for this in the codebase.